### DISCUSSION TITLE
osu!std automatically opening .osz files upon download

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#20176] [Q&A]
Is there a way to disable `.osz` files automatically opening in STANDARD upon download? this makes it very difficult to get maps into osu!lazer, as the files disappear immediately upon being opened in standard.

**Comment by peppy** [comment]
Do you mean "stable" instead of "standard"? If so, this is being tracked at https://github.com/ppy/osu/issues/14824 (which will allow lazer to overwrite the associations that stable has in place).

**Reply by Axilotl17** [reply]
yes sorry, got it.
is there a temporary fix we can use to just import files to lazer?

**Reply by peppy** [reply]
Drag file into window instead?

**Reply by nagi-desuuu** [reply]
There's also the "Import Files" option in the settings, but it's a little less convenient than just dragging the file to lazer, as peppy suggested.

**Reply by Axilotl17** [reply]
> Drag file into window instead?

No, like, IMMEDIATELY upon download stable automatically opens. there is no option to put it in lazer. the file is not accessible after download as it is in stable and has been moved.

I do not believe it is a practical solution to import maps to lazer every time, plus, since sliders are different in lazer, when they're imported to stable it ruins the `.osu`, and then when imported to lazer they're still broken

**Reply by peppy** [reply]
stable does not "ruin" anything.

see the linked issue for a final solution, which will be implemented when it gets implemented. i've given you alternatives you can work with so take them or leave.

**Reply by Axilotl17** [reply]
> stable does not "ruin" anything.

import a lazer map with perfect curves into stable, then go into lazer and do "import beatmaps from stable"
it will break.
